app/views/notifications/_aggregated_reactions.html.erb

```diff
@@ -22,7 +22,7 @@
     reacted to
     <a href="<%= reactable_data["path"] %>" class="notification-comment-reacted-link">
       <%# your article/comment or the actual title of the article/comment %>
-      <%= reactable_data["title"].blank? ? "your #{reactable_data['class']['name'].downcase}" : sanitize(reactable_data["title"]) %>
+      <%= reactable_data["title"].blank? ? "your #{reactable_data['class']['name'].downcase}" : h(reactable_data["title"]) %>
```

Comment:
Where does `h` come from?

Reply:
It's an alias for 'html_escape' defined inside the Active Support Extension. 

https://edgeapi.rubyonrails.org/classes/ERB/Util.html#method-c-h